basics/test.ts

```diff
@@ -0,0 +1,52 @@
+import * as Kilt from "@kiltprotocol/sdk-js";
+import { Keyring } from "@kiltprotocol/utils";
+import { verifyDid } from "./addVerification2Did";
+import { generateAccounts } from "./generateAccount";
+import { generateDid } from "./generateDid";
+import { issueCredential } from "./issueCredential";
+
+async function runAll(): Promise<void> {
+  let api = await Kilt.connect("wss://peregrine.kilt.io/");
+
+  console.log("connected");
+
+  const faucetAccount = Kilt.generateKeypair({
+    type: "ed25519",
+    seed: "0xe566520fec3ca23d80dfe9e9529ada463b93fc33f17219c1089de906f7253f1c",
+  });
+
+  // Yeni bir Keyring örneği oluştur
+  const keyring = new Keyring({ type: "ed25519" });
+
+  // Seed'den KeyringPair oluştur
+  const keyringPair = keyring.addFromSeed(
+    Buffer.from(
+      "e566520fec3ca23d80dfe9e9529ada463b93fc33f17219c1089de906f7253f1c",
+      "hex"
+    )
+  );
+
+  let { issuerAccount, submitterAccount, holderAccount, verifierAccount } =
```

Comment:
Depending on how many accounts you'd actually setup. It might be useful to make a separate file that handles the account creation and funding. Similar to how we have handled it in the past. 

For now this is good.